x/staking/types/delegation_test.go

```diff
@@ -75,3 +77,65 @@ func TestRedelegationString(t *testing.T) {
 
 	require.NotEmpty(t, r.String())
 }
+
+func TestDelegationResponses(t *testing.T) {
```

Comment:
we should eventually do a refactor to create table-based tests

Reply:
IMHO there isn't enough to test here to be table driven.